package.json

```diff
@@ -95,5 +96,8 @@
     "url-loader": "^0.5.7",
     "watch": "^0.16.0",
     "webpack": "^1.13.2"
+  },
+  "devDependencies": {
```

Comment:
We;re currently keeping everything as regular dependencies, no dev deps (although I don't think I agree with that logic). Regardless, if we put this up to run on a server, shouldn't it be a regular dep?